connect.go

```diff
@@ -53,6 +54,20 @@ const (
 	StreamTypeBidi              = StreamTypeClient | StreamTypeServer
 )
 
+func (s StreamType) String() string {
+	switch s {
+	case StreamTypeUnary:
+		return "unary"
+	case StreamTypeClient:
+		return "client"
+	case StreamTypeServer:
+		return "server"
+	case StreamTypeBidi:
+		return "bidi"
+	}
+	return fmt.Sprintf("stream_%d", s)
```

Comment:
If we do do this, generally we've just done `strconv.Itoa(int(s))` for these cases